### PR TITLE
[RPS-531] Add attachments and links to series

### PR DESCRIPTION
**Pull request by jacsamell**
Also added taxonomy to the schema.org markup for publications and
datasets.

### CODE REVIEW

File: repository-data/webfiles/src/main/resources/site/freemarker/include/imports.ftl [hunk 1]
```diff
@@ -8,6 +8,7 @@
 <#assign formatRestrictableDate="uk.nhs.digital.ps.directives.RestrictableDateFormatterDirective"?new() />
 <#assign formatCoverageDates="uk.nhs.digital.ps.directives.CoverageDatesFormatterDirective"?new() >
 <#assign formatDate="uk.nhs.digital.ps.directives.DateFormatterDirective"?new() >
+<#assign formatFileSize="uk.nhs.digital.ps.directives.FileSizeFormatterDirective"?new() />
```
Comment (ngta1) [inline repository-data/webfiles/src/main/resources/site/freemarker/include/imports.ftl]: 👍

File: site/src/main/java/uk/nhs/digital/ps/beans/Dataset.java [hunk 2]
```diff
@@ -89,10 +89,10 @@ public Publication getParentPublication() {
         return HippoBeanHelper.getParentPublication(this);
     }
 
-    public List<List<String>> getTaxonomyList() {
+    public String getFullTaxonomyList() {
         assertPropertyPermitted(PropertyKeys.TAXONOMY);
 
-        return HippoBeanHelper.getTaxonomyList(getKeys());
+        return String.join(",", HippoBeanHelper.getFullTaxonomyList(this));
```
Comment (ziemsky) [inline site/src/main/java/uk/nhs/digital/ps/beans/Dataset.java]: While the Dataset bean is a model for view, emitting a comma-separated string from here feels like tying the model a bit too closely to just one form of presentation. Should the same list of taxonomy values be needed to be displayed using some more elaborate HTML structure at some point, we'd need either to add a new method here, laboriously break the comma-separated string into a list in the template or re-work this method to return a list once again.

Could the method continue to return a list but delegate formatting to Freemarker? I haven't tried it, but [join](https://freemarker.apache.org/docs/ref_builtins_sequence.html#ref_builtin_join) built-in looks like a promising candidate.

File: site/src/main/java/uk/nhs/digital/ps/beans/PublicationBase.java [hunk 3]
```diff
@@ -115,6 +109,12 @@ public HippoBean getParentDocument() {
         return getPropertyIfPermitted(PropertyKeys.INFORMATION_TYPE);
     }
 
+    public String getFullTaxonomyList() {
+        assertPropertyPermitted(Dataset.PropertyKeys.TAXONOMY);
+
+        return String.join(",", HippoBeanHelper.getFullTaxonomyList(this));
```
Comment (ziemsky) [inline site/src/main/java/uk/nhs/digital/ps/beans/PublicationBase.java]: See similar comment for `Dataset` regarding delegating the 'join' operation to Freemarker's [join](https://freemarker.apache.org/docs/ref_builtins_sequence.html#ref_builtin_join) built-in.

File: site/src/main/java/uk/nhs/digital/ps/beans/Dataset.java (hunk 2 above)
Comment (jacsamell) [reply site/src/main/java/uk/nhs/digital/ps/beans/Dataset.java]: Good suggestion, have updated

File: site/src/main/java/uk/nhs/digital/ps/beans/PublicationBase.java (hunk 3 above)
Comment (jacsamell) [reply site/src/main/java/uk/nhs/digital/ps/beans/PublicationBase.java]: Have updated

**Pull request merged**